src/talks_tweeter_sup.erl

```diff
@@ -48,8 +48,11 @@ init([]) ->
                        type => worker,
                        modules => [tt_importer]
                       },
+
+    ActionInterval = {0, 0, 1},
+    TimeWindow = {0, 0, 3},
     SchedulerWorker = #{id => tt_scheduler,
-                        start => {tt_scheduler, start_link, []},
+                        start => {tt_scheduler, start_link, [ActionInterval, TimeWindow]},
```

Comment:
This options, `ActionInterval` and `TimeWindows` should be put into the [config/sys.config](https://github.com/lambdaacademy/2015.12_erlang/blob/master/config/sys.config) file as following:

```erlang
[
 {talks_tweeter, [{action_interval, {0,0,1},
                          {time_window, {0,0,3}]},           
{etweet, [
             ....
             ]
 }].
````

Then, this file is read upon the start up (see [here](https://github.com/lambdaacademy/2015.12_erlang/blob/master/Makefile#L35) and the environment variables can be accessed via `application:get_env/3`, for example:

```erlang
ActionInterval = application:get_env(_AppName = talks_tweeter, _Key = action_interval, _Default = {0,0,1}),
TimeWindow = application:get_env(talks_tweeter, time_window, {0,0,3}).
```

So let's do it the right way.